v2/inc/c_logging/log_context_property_basic_types.h

```diff
@@ -25,7 +27,9 @@
     uint8_t
 
 #define DECLARE_BASIC_TYPE(type_name) \
-    extern const LOG_CONTEXT_PROPERTY_TYPE_IF MU_C2(type_name, _log_context_property_type);
+    int LOG_CONTEXT_PROPERTY_TYPE_INIT(type_name)(void* dst_value, type_name src_value); \
```

Comment:
>void [](http://example.com/codeflow?start=50&length=4)

isn't this "type_name"?

Reply:
I'll leave this is as void* and if possible to solve the alignment later then we can type it.